modules/visual_script/visual_script_editor.cpp

```diff
@@ -527,6 +523,25 @@ void VisualScriptEditor::_update_graph(int p_only_id) {
 			gnode->connect("resize_request", this, "_comment_node_resized", varray(E->get()));
 		}
 
+		if (node_styles.has(node->get_category())) {
+			Ref<StyleBoxFlat> sbf = node_styles[node->get_category()];
+			if (gnode->is_comment())
+				sbf = EditorNode::get_singleton()->get_theme_base()->get_theme()->get_stylebox("comment", "GraphNode");
+
+			Color c = sbf->get_border_color(MARGIN_TOP);
+			c.a = 1;
+			if (EditorSettings::get_singleton()->get("interface/theme/use_graph_node_headers")) {
+				Color mono_color = ((c.r + c.g + c.b) / 3) < 0.5 ? Color(1.0, 1.0, 1.0) : Color(0, 0, 0);
+				mono_color.a = 0.85;
+				c = mono_color;
+			}
+
+			gnode->add_color_override("title_color", c);
+			c.a = 0.7;
+			gnode->add_color_override("close_color", c);
+			gnode->add_style_override("frame", sbf);
+		}
```

Comment:
@djrm There should be an `else` here with a default style. Custom nodes can specify their own category and by default the title is black, which is invisible with the default dark theme.

Ideally, there should be some way to add custom colors for custom categories too.

Reply:
@vnen oh yeah saw a report for that, i will fix it.